WalletWasabi.Tests/UnitTests/Bases/LastExceptionTrackerTests.cs

```diff
@@ -0,0 +1,52 @@
+#nullable enable
```

Comment:
As discussed a while ago, we should turn it on solution level and not trash files with this.

Reply:
The idea is simply add this line, then when NRT are merged, we can remove these lines. It's not that controversial.